examples/0_Simple/simpleStreams/src/host/main.adb

```diff
@@ -101,8 +94,9 @@ begin
 
    pragma CUDA_Execute
      (Init_Array
-        (Uncheck_Convert (Device_A),
-         Uncheck_Convert (Device_C),
+        (Device_A,
+         0, N - 1,
+         Device_C,
          N_Iterations),
```

Comment:
Presented this way, I can hardly decipher the arguments. I do not understand the 'full line' aversion of GnatStudio as it really does not generate clearer code.